accounts/delete.php

```diff
@@ -0,0 +1,46 @@
+<?php
+require_once  '../includes/db/db.php'; //database conn global file
+/** @var mysqli $db */
+$defaultError = 'Account delete unsuccessful';
+
+$userId = testInput('php://input', 'userId');
+
+//checks if userId was sent, checks if its integer-only
+if(isset($userId) && ctype_digit($userId)) {
+
+    //removes column from users table based on given id, 
+    //and also deletes product_user columns associated with said user
+    $sql = sprintf(
+        "DELETE FROM users INNER JOIN product_user WHERE users.id= product_user.user_id and users.id=$userId");
```

Comment:
Om het veilig te maken is het handig om niet direct variabelen in een query te zetten...
Bijv: van:  "DELETE FROM users INNER JOIN product_user WHERE users.id= product_user.user_id and users.id=$userId");
Naar:  "DELETE FROM users INNER JOIN product_user WHERE users.id= product_user.user_id and users.id='$userId'");